vss-cats/src/main/scala/com/virtuslab/vss/cats/base/http/BaseHttpServer.scala

```diff
@@ -0,0 +1,23 @@
+package com.virtuslab.vss.cats.base.http
+
+import org.http4s.HttpApp
+import cats.effect.kernel.Resource
+import com.comcast.ip4s._
+import org.http4s.server.Server
+import cats.effect.kernel.Async
+import org.http4s.ember.server.EmberServerBuilder
+import org.typelevel.log4cats.Logger
+import com.virtuslab.vss.cats.base.config.BaseAppConfig
+
+object BaseHttpServer:
+  def printSwaggerMessage[F[_]: Logger](server: Server): F[Unit] =
+    Logger[F].info(s"Go to http:/${server.address}/docs to open SwaggerUI for the Base service.")
+
+  def make[F[_]: Async: Logger](appConfig: BaseAppConfig, app: HttpApp[F]): Resource[F, Server] =
+    EmberServerBuilder
+      .default[F]
+      .withHostOption(Host.fromString(appConfig.httpHost))
+      .withPort(Port.fromInt(appConfig.httpPort).getOrElse(port"8080"))
```

Comment:
`Port.fromInt` returns None if ports is outside the range 0-65535. I think it's better to throw an exception and blow up if port is invalid instead of using default one, because it means something is wrong with the config.